### DISCUSSION TITLE
Enabling PHP Pint

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LunarVim/LunarVim#4259] [Q&A] [answered]
I installed [pint ](https://laravel.com/docs/10.x/pint) via Mason and I'm expecting it to run automatically when on a .php file but it isn't enabling.

![image](https://github.com/LunarVim/LunarVim/assets/51720003/c4399e55-d89d-4af5-bd22-708c6ae9b592)

I reset the cache after installing the formatter. Do I need to do anything else to enable it?

**Comment by juicy-g** [comment, marked as answer]
Have you tried setting up pint in your config?

```lua
local formatters = require "lvim.lsp.null-ls.formatters"
formatters.setup {
  { name = "pint" },
}
```

**Reply by RizaHKhan** [reply]
OK, this works, but, it gets enabled for every file. I want it to only start for PHP files. Would you know how to achieve that?

**Reply by juicy-g** [reply]
Try this:
```lua
local formatters = require "lvim.lsp.null-ls.formatters"
formatters.setup {
  { 
    name = "pint",
    filetypes = { "php"}
  }
}
```
